### DISCUSSION TITLE
Place the Pop-up outside the Destination Text view

### DISCUSSION THREAD

**Discussion by SalitaNgDiyos** [rubiot/ibiblia#83] [Ideas]
Is it possible to move the pop-up window outside the destination text section? The reason being is that while the pop up definition is really needed during translation, yet when I am trying to associated multiple words at the same time, the pop-up screen blocks part, sometimes all, of the words I need to associate unless I disable the pop up definitions. I will try to use the Alt+hover option and see if this will work out fine. 

Update: Still would prefer pop up dictionary using "mouse hover" only (best for translation projects)

![image](https://user-images.githubusercontent.com/46296587/103432659-c94d9700-4c1d-11eb-8e0a-d57f4b2f3750.png)

**Comment by rubiot** [comment]
I think I solved these unsolicited popups once and for all. I changed the logic to trigger the popup only when the trigger key is pressed **alone**. This means that the definition won't popup anymore if Shift or Ctrl+Shift is pressed.

**Reply by SalitaNgDiyos** [reply]
"...the trigger key is pressed alone..." or just the mouse hover, right? Coz "mouse hover" is what I use all the time. Hehe.

**Reply by rubiot** [reply]
Yes, mouse over will not trigger the popup if another key is pressed. Let me know if this solves this problem for good.

**Reply by SalitaNgDiyos** [reply]
Working perfectly! Thanks for the fix. No more catching flies!

**Reply by rubiot** [reply]
Good to know.

**Comment by rubiot** [comment]
This will be available soon in the next release.
